website/content/docs/configuration/time-format.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: Time String Format
+description: A description of Vault's time string format used throughout its configuration.
+---
+
+# Time String Format
+
+Vault uses a unique but ultimately simple time string format as part of its configuration and APIs.
+This format is used throughout Vault wherever time units, such as durations, need to be given. For
```

Comment:
In some places we reference https://golang.org/pkg/time/#ParseDuration, which is almost what Vault does (the differences IIRC are that we also support the `d` unit, and we assume no unit means seconds.) 

I suspect some of those references to ParseDuration are incorrect and we actually support "vault style durations" there.  There do seem to be a handful where we really are only support Go-style durations though, e.g. pg & mssql & mysql's ttls, pki crls, which may be bugs.  And then command-line flags that take durations are a hybrid, where we treat no-unit as seconds but we don't support the 'd' unit... which is probably also a bug.

Originally I was going to say that you should qualify that "wherever", but I think this exceptions are few enough and probably fixable that I guess we can let it stand.  Instead, do you want to search for references we make to the Go ParseDuration in mdx files and point them to your doc?

Reply:
Agreed - I think it makes sense to consider inconsistencies as bugs until proven otherwise.

After this initial review, I was planning to go through and update all of the many, many pages that reference this - so updating anywhere that references ParseDuration would definitely be included, but thanks for calling it out!